WISE-Community/WISE
# Removed sticky site header and added a "Back to Top" button

Generally following Neilsen/Norman guidelines for back to top buttons: https://www.nngroup.com/articles/back-to-top/.

To test:
- Load any site pages and scroll down at least 360px. Header and top banner should scroll with page.
- Start scrolling up. "Back to Top" button should appear fixed in the bottom right corner as you scroll (until page scroll is less than 360px).
- Press "Back to Top" button. Screen should jump to the top.

Resolves #1997.
